### PR TITLE
Log AccessControl invocations

### PR DESCRIPTION
**Pull request by kokosing**
Log AccessControl invocations

### CODE REVIEW

File: presto-main/src/main/java/io/prestosql/security/AccessControlModule.java [hunk 1]
```diff
@@ -26,7 +31,24 @@
...
         newExporter(binder).export(AccessControlManager.class).withGeneratedName();
     }
+
+    @Provides
+    @Singleton
+    public AccessControl createAccessControl(AccessControlManager accessControlManager)
+    {
+        Logger logger = Logger.get(AccessControl.class);
+
+        // TODO enable logging invocation when logging configuration was changed at the runtime
```
Comment (findepi) [inline presto-main/src/main/java/io/prestosql/security/AccessControlModule.java]: what do we do with this?

Comment (kokosing) [reply presto-main/src/main/java/io/prestosql/security/AccessControlModule.java]: IMO it requires to implement `ForwardingAccessControl` it has about 20 methods to implement. I was too lazy for that today. Maybe it is great `good first issue` for somebody else.

Other than that I don't think many users know how to change logging configuration at runtime.

Comment (sopel39) [reply presto-main/src/main/java/io/prestosql/security/AccessControlModule.java]: > it has about 20 methods to implement.

That doesn't seem like a lot (just a bit of copy-paste). Should we address it in this PR too?

Comment (kokosing) [reply presto-main/src/main/java/io/prestosql/security/AccessControlModule.java]: It took me about 2h :)

**Comment by kokosing** [comment]
Comments addressed.

**Pull request merged**